app/controllers/api/databases.php

```diff
@@ -4157,18 +4419,20 @@ function updateAttribute(
         }
 
         try {
-            $dbForProject->withRequestTimestamp($requestTimestamp, function () use ($dbForProject, $database, $collection, $documentId) {
-                $dbForProject->deleteDocument(
-                    'database_' . $database->getInternalId() . '_collection_' . $collection->getInternalId(),
-                    $documentId
-                );
-            });
+            $dbForProject->deleteDocument(
+                'database_' . $database->getInternalId() . '_collection_' . $collection->getInternalId(),
+                $documentId
+            );
         } catch (NotFoundException $e) {
             throw new Exception(Exception::COLLECTION_NOT_FOUND);
         }
 
+        $operations = 0;
+
         // Add $collectionId and $databaseId for all documents
-        $processDocument = function (Document $collection, Document $document) use (&$processDocument, $dbForProject, $database) {
+        $processDocument = function (Document $collection, Document $document) use (&$processDocument, $dbForProject, $database, &$operations) {
```

Comment:
Is this missing on production? We might need to share a changelog update if this is the case.

Reply:
We also need to make sure the min count for each operation is 1, even if no document was impacted.

Reply:
Yeah it's missing on prod, will add a PR note that it should be in changelog.

> We also need to make sure the min count for each operation is 1, even if no document was impacted.
Updated to enforce min 1